cli-config/src/config.rs

```diff
@@ -12,13 +22,35 @@ lazy_static! {
     };
 }
 
+/// The Solana CLI configuration.
 #[derive(Serialize, Deserialize, Debug, PartialEq)]
 pub struct Config {
+    /// The RPC address of a Solana validator node.
+    ///
+    /// Typical values for mainnet, devnet, and testnet are [described in the
+    /// Solana documentation][rpcdocs].
+    ///
+    /// For local testing, the typical value is `http://localhost:8899`.
+    ///
+    /// [rpcdocs]: https://docs.solana.com/cluster/rpc-endpoints
     pub json_rpc_url: String,
+    /// The address to connect to for receiving event notifications.
+    ///
+    /// If it is an empty string then the correct value will be derived
+    /// from `json_rpc_url`.
+    ///
+    /// The default value is the empty string.
     pub websocket_url: String,
+    /// The path to the keypair file.
     pub keypair_path: String,
+    /// A mapping from Solana addresses to human-readable names.
+    ///
+    /// By default the only value in this map is the system program.
     #[serde(default)]
     pub address_labels: HashMap<String, String>,
+    /// The default commitment level.
+    ///
+    /// By default the value is "confirmed".
```

Comment:
Maybe link to solana-sdk::commitment-config::CommitmentLevel::Confirmed ?

Reply:
Will do.